testFramework/com/twitter/intellij/pants/testFramework/PantsIntegrationTestCase.java

```diff
@@ -495,6 +499,21 @@ public void tearDown() throws Exception {
       killNailgun();
       cleanProjectRoot();
       Messages.setTestDialog(TestDialog.DEFAULT);
+
+      // TODO(dmcclanahan): after updating from 172.4343.14 to 173.3531.6,
+      // intellij's provided test class sometimes yells about a leaky jdk
+      // table. i don't think this indicates any problems, so for now if tests
+      // fail with leaky sdk errors, broaden this to include the leaked sdks
```

Comment:
does the comment and following code still apply assuming you fixed it with passing in `getTestRootDisposable()`?

Reply:
Unfortunately, yes -- the error that is fixed by using `getTestRootDisposable` is not the same as the leaky jdk table error (says "not disposed", while the leaky jdk table error message does not mention disposing of anything). I have not been able to understand why this change in behavior occurs yet.